sources.go

```diff
@@ -78,6 +81,7 @@ func FindFiles(path string) *Pipe {
 // supplied path, one per line. The path may be a glob, conforming to
 // filepath.Match syntax.
 func ListFiles(path string) *Pipe {
+	path = strings.TrimRight(path, "\n")
```

Comment:
Same applies here; if `ListFiles` is not a filter, it doesn't take any pipe input, so it doesn't need any trimming. If it _were_ a filter, I suspect it wouldn't need trimming either; but we can find out.